mkdocs.yml

```diff
@@ -27,14 +68,16 @@ extra_css:
 copyright: '&copy; 2020, DietPi'
 
 nav:
-  - 'Overview' : index.md
-  - 'Installation' : 'user-guide_install.md'
-  - 'Getting started': 
-      - 'Overview' : 'user-guide_overview.md'
+  - 'Home' : 
+      - 'Overview' : index.md
+      - 'Installation' : 'user-guide_install.md'
+      - 'Getting started' : 'user-guide_overview.md'
       - 'DietPi Tools' : 'user-guide_tools.md'
-  - 'Optimised software' : 'user-optimised-software.md'
-  - 'Supported SBCs' : 'hardware-supported_sbc.md'
-  - 'Updates' : 'release-notes-updates.md'
-  - 'Community' : 'https://dietpi.com/phpbb/index.php'
-  - 'Request features' : 'https://github.com/MichaIng/DietPi/issues/new/choose'
-  - 'Report issues' : 'https://github.com/MichaIng/DietPi/issues/new?template=bug_report.md'
+      - 'Optimised software' : 'user-optimised-software.md'
+      - 'Supported SBC' : 'hardware-supported_sbc.md'
+  - 'Updates' : 
```

Comment:
I just tested this and was wondering where "Updates" and "Community" was gone. Took a while until I found the register at the top. Not sure if this is only as I expected them to be in left side navigation or if other users might oversee that as well?